main.go

```diff
@@ -118,6 +121,8 @@ func argsParse(args []string) (runOptionsType, error) {
 
 	daemon := parsing.Bool("daemon", false, "Run as a daemon.")
 
+	updateCheck := parsing.Bool("check-update", false, "force update check")
```

Comment:
I think this flag is important enough that it deserves an integration test. Just clone an existing test, set the polling interval to be really long, and then use this flag to speed it up.